debian/rules

```diff
@@ -78,7 +78,7 @@ override_dh_auto_clean:
 override_dh_auto_configure:
 	# Run qmake once to create .qmake.conf and be sure to append the following values.
 	qmake QT_BUILD_PARTS+=tests QMAKE_EXTRA_ARGS+="$(config_args)"
-	# Rebuild the SyncQt header files
+	# Rebuild the SyncQt header files - update this on package updates
 	rm -rf include/
 	/usr/lib/$(DEB_HOST_MULTIARCH)/qt5/bin/syncqt.pl -version 5.11.2
```

Comment:
You may want to replace hard-coded `5.11.2` with something like `$(DEB_VERSION_UPSTREAM:+dfsg=)`.